sonar-ballerina-plugin/src/main/resources/io/ballerina/sonar/ballerina-5.html

```diff
@@ -0,0 +1,45 @@
+<p>
+    An class defined as isolated is similar to a module with isolated module-level variables. Mutable fields of an isolated object,
+</p>
+<ul>
+    <li>must be private and so can only be accessed using self</li>
+    <li>must only be accessed within a lock statement</li>
```

Comment:
This is same as isolated variables

Imo there are three options
1. final radonly
2. isolatd field
3. private

Reply:
Resolved with f10b04b7337bff186e2de3ae2a65a2421c10884d